### DISCUSSION TITLE
Why isn't there an "action_m" in the config?

### DISCUSSION THREAD

**Discussion by jamieburchell** [fail2ban/fail2ban#3757] [General]
My fail2ban emails have "missing whois program" at the end of them. I know why (it's not installed). I don't really want the Whois info so I was looking to remove it from the email when I discovered there's no option to out of the box as far as I can tell.

There's `action_`, `action_mw` and `action_mwl` but curiously no `action_m` for just sending an email with no Whois. 

There is however a `sendmail.conf`.

I'm assuming it's simple enough to create:

```
# ban & send an e-mail to the destemail.
action_m = %(action_)s
           %(mta)s[sender="%(sender)s", dest="%(destemail)s", protocol="%(protocol)s", chain="%(chain)s"]
```

I'm just curious as to why it's missing? 🤔

**Comment by sebres** [comment]
> `%(mta)[sender="%(sender)s", dest="%(destemail)s", protocol="%(protocol)s", chain="%(chain)s"]`

Guess this action would not need protocol and chain.

> I'm just curious as to why it's missing?

Because (as you wrote) it is simple enough to create it (or specify 2nd action directly in `action`)... 
All that substitutions are just examples, let alone the necessity to get a mail for each ban.
